packages/calypso-build/webpack/util.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Transform webpack output.filename and output.chunkFilename to CSS variants
+ *
+ * @param {(string|undefined)} name filename, chunkFilename or undefined
+ * @return {(string|undefined)}     Transformed name or undefined
+ */
+function cssNameFromFilename( name ) {
+	if ( name ) {
+		const [ cssChunkFilename, chunkQueryString ] = name.split( '?', 2 );
+		return cssChunkFilename.replace(
```

Comment:
If there's a query string but the extension isn't `.js`, we return the input without the query:

```
x.ts?query
x.ts
```

That's buggy. Maybe we should throw and error if it doesn't match `/\.js$/i`